src/Microsoft.ML.Data/DataView/LambdaFilter.cs

```diff
@@ -1,194 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Reflection;
-using Microsoft.Data.DataView;
-using Microsoft.ML.Data.Conversion;
-using Microsoft.ML.Model;
-
-namespace Microsoft.ML.Data
-{
-    /// <summary>
-    /// This applies the user provided RefPredicate to a column and drops rows that map to false. It automatically
-    /// injects a standard conversion from the actual type of the source column to typeSrc (if needed).
-    /// </summary>
-    public static class LambdaFilter
```

Comment:
>LambdaFilter [](start = 24, length = 12)

I don't think we should delete this. While I doubt we'll get around to this for v1, I was planning on using this to solve #1763.

Reply:
Ok, thanks. I will revert it.

Reply:
All right, can still be internal BTW till we work out its public surface, but I'd rather not delete.